en/docs/deploy-and-publish/publish-on-dev-portal/publish-aws-apis-in-the-dev-portal.md

```diff
@@ -0,0 +1,118 @@
+    Open README file to learn more
+    ```
+### How the **aws init** command works.
+
+The `aws init` command will be using the AWS CLI in the background to get a list of all the APIs from the AWS API gateway. It will then look for the ID of the API name provided by the user when the `aws init` command was executed, to download the OAS of that API. After making the necessary adjustments to the APIs version, tags, endpoints etc, an API project will be initialized and the downloaded OAS will be deleted.
+
+### Importing the initialized API project to the WSO2 API Manager as an API
+
+Upon successful initialization of the project, execute the following command to import that API project to the WSO2 publisher portal as an API. 
+
+-   **Command**
+
+    ``` bash
+    apictl import api -f <path-to-api-project> -e <environment> -k
+    ```
+
+!!! example
+    ```bash
+    apictl import api -f /home/user/Documents/product-apim-tooling/import-export-cli/products -e production -k
+    ```
+
+After successfully importing the API, it will appear in the publisher as shown below.
+
+[![]({{base_path}}/assets/img/publish/aws-api-publisher.png)]({{base_path}}/assets/img/publish/aws-api-publisher.png)
+
+To publish the API, go to `Lifecycle` under `Publish` and click on `Publish`.
+
+[![]({{base_path}}/assets/img/publish/publish-aws-api.png)]({{base_path}}/assets/img/publish/publish-aws-api.png)
+
+To learn more about importing an API project refer to [import-api-project](https://apim.docs.wso2.com/en/latest/learn/api-controller/importing-apis-via-dev-first-approach/#import-an-api-project).
+
+The initial state of the API will be set to `CREATED` so the user will be able to make changes to the API if necessary in the publisher portal, before publishing the API on the developer portal.
+
+## Troubleshooting errors
+
+If any errors are being printed and if the problem is not clear, execute the `aws init` command with the `--verbose` flag to get a more detailed output.
+
+### Unable to find an API with the name `<api-name>`
+
+After making sure the user has provided the correct API name and that the API is in the AWS API gateway, if the problem persists, it is possible that the user has disabled AWS CLI pagination which will cause the AWS CLI to output only a limited number of APIs. In such cases, the API the user is looking for might not be in the APIs list provided by the AWS CLI. Enable AWS CLI pagination and try again. 
+
+### Error code 255
+
+One of the most common errors related to this command is the `exit status 255`. This is a AWS CLI error code and it means the AWS CLI command failed and that there were errors generated by the AWS CLI or by the AWS service to which the request was sent.
+The most common reason for this is due to a incorrect stage name provided.
+
+!!! tip
+    Make sure you provide the correct stage name and the spellings are correct.
+    It is also important to remember the name and stagename are case sensitive.
```

Comment:
```suggestion
    It is also important to remember the `name` and `stage`. `name` are case sensitive.
```

Reply:
@Chamindu36 `name` and `stagename` are two different things. stage-name is the stage of that API.

Reply:
Yes. Isn't it good if we can use the text style (the style used in commands and flags) as we do in the other places in the documentation for these properties without using the normal style

Reply:
yes @Chamindu36 will do that. but i meant the second change you had mentioned ( `stage`, `name`). That is not required right ? because their `stagename` is one.

Reply:
Yes. We can omit that